Add-ons/UmbracoForms/Developer/Custom-Markup/index.md

```diff
@@ -48,7 +48,7 @@ Contents of the `FieldType.Textfield.cshtml` view (from the default theme):
         @{if (Model.Validate) {<text> data-val-regex="@Model.InvalidErrorMessage" data-regex="@Html.Raw(Model.Regex)"</text>}}
     />
 
-By default the form makes use of jQuery validate and jQuery validate unobtrusive. That's why you see attributes like `data-val` and `data-val-required`.
+By default the form makes use of jQuery validate and jQuery validate unobtrusive which is why you see attributes like data-val and data-val-required.
```

Comment:
I'm curious why you removed the formatting of the attribute names? 😁

Reply:
That's very weird, I didn't... at least not intentionally, it looks like it might be a fault with the app I was using to do the editing?! I'll see if I can fix it.